inst/EpiModel2.rmd

```diff
@@ -0,0 +1,684 @@
+
+```{r echo = TRUE, eval = FALSE}
+## pre-EpiModel 2.0 Workflow
+num1 <- num2 <- 500
+nw <- network.initialize(n = num1 + num2, directed = FALSE, bipartite = num1)
+formation <- ~edges
+target.stats <- 400
+```
+
+```{r echo = TRUE, eval = FALSE}
+## EpiModel 2.0 
+num1 <- num2 <- 500
+
+#update to new initialization function network_initialize.
+nw <- network_initialize(n = num1 + num2)
+
+#Update to new vertex assignment function set_vertex_attribute
+nw <- set_vertex_attribute(nw, "group", rep(c(1,2), c(num1,num2)))
+
+#If looking to replicate 'bipartite' network:
+formation <- ~edges + nodematch("group")
+target.stats <- c(400, 0)
+```
+
+```{r echo = TRUE, eval = FALSE}
+coef.diss <- dissolution_coefs(dissolution = ~offset(edges), duration = 20)
+est <- netest(nw, formation, target.stats, coef.diss)
+```
+
+```{r echo = TRUE, eval = FALSE}
+## pre-EpiModel 2.0 Workflow
+param <- param.net(inf.prob = 0.4, inf.prob.m2 = 0.2)
+init <- init.net(i.num = 20, i.num.m2 = 20)
+```
+
+```{r echo = TRUE, eval = FALSE}
+## EpiModel 2.0 Workflow
+#Update all parameters and initial conditions from ".m2" to ".g2" language.
+param <- param.net(inf.prob = 0.4, inf.prob.g2 = 0.2)
+init <- init.net(i.num = 20, i.num.g2 = 20)
+```
+
+```{r echo = TRUE, eval = FALSE}
+## pre-EpiModel 2.0 Workflow
+control <- control.net(type = "SI", nsteps = 100, depend = TRUE, 
+                       save.network = TRUE)
+```
+
+```{r echo = TRUE, eval = FALSE}
+## EpiModel 2.0 Workflow
+#control.net parameter depend updated to resimulate.network; save.network folded
+#into tergmLite.
+control <- control.net(type = "SI", nsteps = 100, resimulate.network = TRUE,
+                       tergmLite = FALSE)
+```
+
+```{r echo = TRUE, eval = FALSE}
+sim <- netsim(est, param, init, control)
+```
+
+## SIR, with vital dynamics
+
+```{r echo = TRUE, eval = FALSE}
+## pre-EpiModel 2.0 Workflow
+num <- 1000
+nw <- network.initialize(num, directed = FALSE)
+```
+
+```{r echo = TRUE, eval = FALSE}
+## EpiModel 2.0 Workflow
+num <- 1000
+nw <- network_initialize(num)
+```
+
+```{r echo = TRUE, eval = FALSE}
+formation <- ~edges
+target.stats <- 400
+coef.diss <- dissolution_coefs(dissolution = ~offset(edges), duration = 20, d.rate = 0.005)
+est <- netest(nw, formation, target.stats, coef.diss)
+```
+
+```{r echo = TRUE, eval = FALSE}
+param <- param.net(inf.prob = 0.4, act.rate = 1, rec.rate = 0.02,
+                   b.rate = 0.005, di.rate = 0.005, ds.rate = 0.005,
+                   dr.rate = 0.005)
+init <- init.net(i.num = 30)
+```
+
+```{r echo = TRUE, eval = FALSE}
+## pre-EpiModel 2.0 Workflow
+control <- control.net(type = "SIR", nsteps = 100, depend = FALSE, 
+                       save.network = FALSE)
+```
+
+```{r echo = TRUE, eval = FALSE}
+## EpiModel 2.0 Workflow
+control <- control.net(type = "SIR", nsteps = 100, resimulate.network = FALSE,
+                       tergmLite = TRUE)
+```
+
+```{r echo = TRUE, eval = FALSE}
+sim <- netsim(est, param, init, control)
```

Comment:
```suggestion
```{r echo = TRUE, eval = FALSE}
# Same for both workflow
sim <- netsim(est, param, init, control)
```